docs/import/color_function_util.js

```diff
@@ -458,3 +458,35 @@ function getColorFunction() {
   const index = getRowIndex(globalTd.parents('tr'));
   return getCurrentLayers()[index]['color-function'];
 }
+
+/**
+ * Gets the linear gradient of the colors for the legend.
+ *
+ * @param {Object} colorFunction color data from the layer
+ * @return {string} the linear gradient
+ */
+function getLinearGradient(colorFunction) {
+  if (!colorFunction) {
+    return '';
+  }
+  const currentStyle = colorFunction['current-style'];
+  let gradientString = 'linear-gradient(to right';
+  switch (currentStyle) {
+    case 0:
+      gradientString += ', white, ' + colorFunction['color'];
+      break;
+    case 1:
+      const colors = [...(new Set(Object.values(colorFunction['colors'])))];
+      const percent = 100 / colors.length;
```

Comment:
In the case that you have a discrete layer but have chosen no colors for it, this results in percent = Infinity. Not awesome, although code tolerates it. Should probably add a check for this.